model/modeljson/internal/span.go

```diff
@@ -22,6 +22,7 @@ type Span struct {
 	Composite           *SpanComposite     `json:"composite,omitempty"`
 	Destination         *SpanDestination   `json:"destination,omitempty"`
 	DB                  *DB                `json:"db,omitempty"`
+	Code                *Code              `json:"code,omitempty"`
```

Comment:
Is this change required? Why are we adding it both here and in `document.go`? Should we be mapping it as `code.stacktrace`, or `span.code.stacktrace`? I would expect the former, since that's what we're planning to do for OTel attributes in the future. Also I don't see any code that sets either modeljson type's Code field from the modelpb type...?

Reply:
I wasn't aware that we plan to keep the `span` namespace, in which case that makes sense to write it to `span.code.stacktrace`.

I don't know why I had to add it to both places, I just followed the instructions to add a new otel field.
From what I understand I have to decide adding this field in either `document.go` or `span.go`, which of the two is preferable ?

Reply:
> I wasn't aware that we plan to keep the span namespace, in which case that makes sense to write it to span.code.stacktrace.

No, the opposite - we will stop nesting so many things under `span` in the future. There will be `span.id` and things like that, but in the future attributes will essentially be written as they come (e.g. the `code.stacktrace` OTel attribute would be mapped as `code.stacktrace` in the ES docs).

> From what I understand I have to decide adding this field in either document.go or span.go, which of the two is preferable ?

`document.go`, so it ends up at the top level